[package] poco/1.9.4: Error with SSL on Windows (maybe relinking libraries)

### Package and Environment Details (include every applicable attribute)
  * Package Name/Version: **poco/1.9.4**
  * Operating System+version: **Windows 10**
  * Compiler+version: **VS 2017 Community**
  * Docker image: **conanio/gcc8**
  * Conan version: **conan 1.21.0**
  * Python version: **Python 3.7.4**


### Conan profile (output of `conan profile show default` or `conan profile show <profile>` if custom profile is in use)
```
include(default)

[settings]
os=Windows
arch=x86_64
compiler=Visual Studio
compiler.version=15
```

### Steps to reproduce (Include if Applicable)

1. `git clone https://github.com/ezored/ezored.git -b xcframework-support`
2. `cd ezored`
3. `python make.py conan install_profiles`
4. `python make.py target windows_app conan`

### Logs (Include/Attach if Applicable)

https://ci.appveyor.com/project/PauloCoutinho/ezored/builds/30131372

<details><summary>Click to expand log</summary>

```
c:\projects\ezored\projects\samples\simple\main.cpp(68): warning C4868: compiler may not enforce left-to-right evaluation order in braced initializer list [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(Context.obj) : error LNK2005: "public: virtual __cdecl Poco::Net::Context::~Context(void)" (??1Context@Net@Poco@@UEAA@XZ) already defined in PocoNetSSLmdd.lib(Context.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(Context.obj) : error LNK2005: "public: void __cdecl Poco::Net::Context::enableExtendedCertificateVerification(bool)" (?enableExtendedCertificateVerification@Context@Net@Poco@@QEAAX_N@Z) already defined in PocoNetSSLmdd.lib(Context.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(X509Certificate.obj) : error LNK2005: "public: __cdecl Poco::Net::X509Certificate::X509Certificate(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (??0X509Certificate@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) already defined in PocoNetSSLmdd.lib(X509Certificate.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(X509Certificate.obj) : error LNK2005: "public: __cdecl Poco::Net::X509Certificate::X509Certificate(class std::basic_istream<char,struct std::char_traits<char> > &)" (??0X509Certificate@Net@Poco@@QEAA@AEAV?$basic_istream@DU?$char_traits@D@std@@@std@@@Z) already defined in PocoNetSSLmdd.lib(X509Certificate.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(X509Certificate.obj) : error LNK2005: "public: __cdecl Poco::Net::X509Certificate::X509Certificate(class Poco::Net::X509Certificate const &)" (??0X509Certificate@Net@Poco@@QEAA@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLManager.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(X509Certificate.obj) : error LNK2005: "public: __cdecl Poco::Net::X509Certificate::~X509Certificate(void)" (??1X509Certificate@Net@Poco@@QEAA@XZ) already defined in PocoNetSSLmdd.lib(X509Certificate.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(X509Certificate.obj) : error LNK2005: "public: bool __cdecl Poco::Net::X509Certificate::verify(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)const " (?verify@X509Certificate@Net@Poco@@QEBA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) already defined in PocoNetSSLmdd.lib(X509Certificate.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(X509Certificate.obj) : error LNK2005: "protected: static bool __cdecl Poco::Net::X509Certificate::containsWildcards(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?containsWildcards@X509Certificate@Net@Poco@@KA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) already defined in PocoNetSSLmdd.lib(X509Certificate.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(X509Certificate.obj) : error LNK2005: "protected: static bool __cdecl Poco::Net::X509Certificate::matchWildcard(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?matchWildcard@X509Certificate@Net@Poco@@KA_NAEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@0@Z) already defined in PocoNetSSLmdd.lib(X509Certificate.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLException::SSLException(int)" (??0SSLException@Net@Poco@@QEAA@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLException::SSLException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0SSLException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLException::SSLException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0SSLException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@0H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLException::SSLException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class Poco::Exception const &,int)" (??0SSLException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBVException@2@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLException::SSLException(class Poco::Net::SSLException const &)" (??0SSLException@Net@Poco@@QEAA@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual __cdecl Poco::Net::SSLException::~SSLException(void)" (??1SSLException@Net@Poco@@UEAA@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: class Poco::Net::SSLException & __cdecl Poco::Net::SSLException::operator=(class Poco::Net::SSLException const &)" (??4SSLException@Net@Poco@@QEAAAEAV012@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::SSLException::name(void)const " (?name@SSLException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::SSLException::className(void)const " (?className@SSLException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual class Poco::Exception * __cdecl Poco::Net::SSLException::clone(void)const " (?clone@SSLException@Net@Poco@@UEBAPEAVException@3@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual void __cdecl Poco::Net::SSLException::rethrow(void)const " (?rethrow@SSLException@Net@Poco@@UEBAXXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLContextException::SSLContextException(int)" (??0SSLContextException@Net@Poco@@QEAA@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLContextException::SSLContextException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0SSLContextException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLContextException::SSLContextException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0SSLContextException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@0H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLContextException::SSLContextException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class Poco::Exception const &,int)" (??0SSLContextException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBVException@2@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLContextException::SSLContextException(class Poco::Net::SSLContextException const &)" (??0SSLContextException@Net@Poco@@QEAA@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual __cdecl Poco::Net::SSLContextException::~SSLContextException(void)" (??1SSLContextException@Net@Poco@@UEAA@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: class Poco::Net::SSLContextException & __cdecl Poco::Net::SSLContextException::operator=(class Poco::Net::SSLContextException const &)" (??4SSLContextException@Net@Poco@@QEAAAEAV012@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::SSLContextException::name(void)const " (?name@SSLContextException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::SSLContextException::className(void)const " (?className@SSLContextException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual class Poco::Exception * __cdecl Poco::Net::SSLContextException::clone(void)const " (?clone@SSLContextException@Net@Poco@@UEBAPEAVException@3@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual void __cdecl Poco::Net::SSLContextException::rethrow(void)const " (?rethrow@SSLContextException@Net@Poco@@UEBAXXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::InvalidCertificateException::InvalidCertificateException(int)" (??0InvalidCertificateException@Net@Poco@@QEAA@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::InvalidCertificateException::InvalidCertificateException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0InvalidCertificateException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::InvalidCertificateException::InvalidCertificateException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0InvalidCertificateException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@0H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::InvalidCertificateException::InvalidCertificateException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class Poco::Exception const &,int)" (??0InvalidCertificateException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBVException@2@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::InvalidCertificateException::InvalidCertificateException(class Poco::Net::InvalidCertificateException const &)" (??0InvalidCertificateException@Net@Poco@@QEAA@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual __cdecl Poco::Net::InvalidCertificateException::~InvalidCertificateException(void)" (??1InvalidCertificateException@Net@Poco@@UEAA@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: class Poco::Net::InvalidCertificateException & __cdecl Poco::Net::InvalidCertificateException::operator=(class Poco::Net::InvalidCertificateException const &)" (??4InvalidCertificateException@Net@Poco@@QEAAAEAV012@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::InvalidCertificateException::name(void)const " (?name@InvalidCertificateException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::InvalidCertificateException::className(void)const " (?className@InvalidCertificateException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual class Poco::Exception * __cdecl Poco::Net::InvalidCertificateException::clone(void)const " (?clone@InvalidCertificateException@Net@Poco@@UEBAPEAVException@3@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual void __cdecl Poco::Net::InvalidCertificateException::rethrow(void)const " (?rethrow@InvalidCertificateException@Net@Poco@@UEBAXXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::CertificateValidationException::CertificateValidationException(int)" (??0CertificateValidationException@Net@Poco@@QEAA@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::CertificateValidationException::CertificateValidationException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0CertificateValidationException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::CertificateValidationException::CertificateValidationException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0CertificateValidationException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@0H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::CertificateValidationException::CertificateValidationException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class Poco::Exception const &,int)" (??0CertificateValidationException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBVException@2@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::CertificateValidationException::CertificateValidationException(class Poco::Net::CertificateValidationException const &)" (??0CertificateValidationException@Net@Poco@@QEAA@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual __cdecl Poco::Net::CertificateValidationException::~CertificateValidationException(void)" (??1CertificateValidationException@Net@Poco@@UEAA@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: class Poco::Net::CertificateValidationException & __cdecl Poco::Net::CertificateValidationException::operator=(class Poco::Net::CertificateValidationException const &)" (??4CertificateValidationException@Net@Poco@@QEAAAEAV012@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::CertificateValidationException::name(void)const " (?name@CertificateValidationException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::CertificateValidationException::className(void)const " (?className@CertificateValidationException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual class Poco::Exception * __cdecl Poco::Net::CertificateValidationException::clone(void)const " (?clone@CertificateValidationException@Net@Poco@@UEBAPEAVException@3@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual void __cdecl Poco::Net::CertificateValidationException::rethrow(void)const " (?rethrow@CertificateValidationException@Net@Poco@@UEBAXXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::SSLConnectionUnexpectedlyClosedException(int)" (??0SSLConnectionUnexpectedlyClosedException@Net@Poco@@QEAA@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::SSLConnectionUnexpectedlyClosedException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0SSLConnectionUnexpectedlyClosedException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::SSLConnectionUnexpectedlyClosedException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,int)" (??0SSLConnectionUnexpectedlyClosedException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@0H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::SSLConnectionUnexpectedlyClosedException(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &,class Poco::Exception const &,int)" (??0SSLConnectionUnexpectedlyClosedException@Net@Poco@@QEAA@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@AEBVException@2@H@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::SSLConnectionUnexpectedlyClosedException(class Poco::Net::SSLConnectionUnexpectedlyClosedException const &)" (??0SSLConnectionUnexpectedlyClosedException@Net@Poco@@QEAA@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::~SSLConnectionUnexpectedlyClosedException(void)" (??1SSLConnectionUnexpectedlyClosedException@Net@Poco@@UEAA@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: class Poco::Net::SSLConnectionUnexpectedlyClosedException & __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::operator=(class Poco::Net::SSLConnectionUnexpectedlyClosedException const &)" (??4SSLConnectionUnexpectedlyClosedException@Net@Poco@@QEAAAEAV012@AEBV012@@Z) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::name(void)const " (?name@SSLConnectionUnexpectedlyClosedException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual char const * __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::className(void)const " (?className@SSLConnectionUnexpectedlyClosedException@Net@Poco@@UEBAPEBDXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual class Poco::Exception * __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::clone(void)const " (?clone@SSLConnectionUnexpectedlyClosedException@Net@Poco@@UEBAPEAVException@3@XZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(SSLException.obj) : error LNK2005: "public: virtual void __cdecl Poco::Net::SSLConnectionUnexpectedlyClosedException::rethrow(void)const " (?rethrow@SSLConnectionUnexpectedlyClosedException@Net@Poco@@UEBAXXZ) already defined in PocoNetSSLmdd.lib(SSLException.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
PocoNetSSLWinmdd.lib(Utility.obj) : error LNK2005: "public: static enum Poco::Net::Context::VerificationMode __cdecl Poco::Net::Utility::convertVerificationMode(class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> > const &)" (?convertVerificationMode@Utility@Net@Poco@@SA?AW4VerificationMode@Context@23@AEBV?$basic_string@DU?$char_traits@D@std@@V?$allocator@D@2@@std@@@Z) already defined in PocoNetSSLmdd.lib(Utility.obj) [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
C:\projects\ezored\build\windows_app\Debug\x86_64\target\bin\Sample.exe : fatal error LNK1169: one or more multiply defined symbols found [C:\projects\ezored\build\windows_app\Debug\x86_64\target\Sample.vcxproj]
ERROR: conanfile.py (windows_app/1.0.0): Error in build() method, line 40
	cmake.build()
	ConanException: Error 1 while executing cmake --build C:\projects\ezored\build\windows_app\Debug\x86_64\target --config Debug -- /m:2 /verbosity:minimal
COMMAND: conan build C:\projects\ezored\files\targets\windows_app\conan\recipe\conanfile.py --source-folder C:\projects\ezored\files\targets\windows_app\cmake --build-folder C:\projects\ezored\build\windows_app\Debug\x86_64\target --install-folder C:\projects\ezored\build\windows_app\Debug\x86_64\conan
WORKING DIR: C:\projects\ezored\build\windows_app\Debug\x86_64\target
[ERROR] Command execution has failed
Command exited with code 10
```

</details>

--- a/recipes/poco/all/conanfile.py
+++ b/recipes/poco/all/conanfile.py
@@ -170,8 +170,12 @@ class PocoConan(ConanFile):
                  else ("d" if self.settings.build_type=="Debug" else "")
         for flag, lib in libs:
             if getattr(self.options, flag):
+                if self.settings.os == "Windows" and flag == "enable_netssl" and self.options.enable_netssl_win:
+                    continue
+
                 if self.settings.os != "Windows" and flag == "enable_netssl_win":
                     continue
+
                 self.cpp_info.libs.append("%s%s" % (lib, suffix))
 
         self.cpp_info.libs.append("PocoFoundation%s" % suffix)

